modules/spring-config/src/main/resources/minimal-mule-config.xml

```diff
@@ -18,30 +16,57 @@
 
         <spring:bean name="_defaultPersistentQueueStore,_fileQueueStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultPersistentQueueStore"/>
 
-        <spring:bean name="_defaultInMemoryObjectStore" class="org.mule.util.store.DefaultObjectStoreFactoryBean" factory-method="createDefaultInMemoryObjectStore"/>
+        <!--=================================-->
+        <!--BEGINNING OF LOCAL/CLUSTER SECTION-->
+        <!--=================================-->
 
-        <spring:alias name="_defaultInMemoryObjectStore" alias="_localInMemoryObjectStore" />
+        <!--The following section contains beans which implementation is replaced when the cluster core extension is active.-->
```

Comment:
hurst nobody and makes for better documentation. I wish there were a CE example but there isn't